app/views/users/change_password.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Change Password</h1>
+
+<%= form_for @user do |f| %>
```

Comment:
It's best to `simple_form`as you've been using it elsewhere for the same of consistency.